### PR TITLE
Warn about outdated browsers

### PR DESCRIPTION
**Pull request by marcelgerber**
This PR shows a Foundation alert if an old/outdated browser was detected.

For screens (@larz0) see http://www.browserstack.com/screenshots/3ac65841a4b0189d2b9b1671933b955002b291fc
According to the screens, the alert is shown for:
- IE <= 9
- Firefox <= 4
- Opera <= 11.6

Don't know exactly 'bout Chrome and Safari.

### CODE REVIEW

File: contribute.html [hunk 1]
```diff
@@ -108,6 +108,10 @@ <h1 class="brackets-logo">
         -->
 
     <div id="hero-wrapper" class="gradient">
+        <div id="browser-alert" class="alert-box warning" data-alert>
+            Looks like you're using an outdated browser. We strongly recommend updating it or <a href="http://www.whatbrowser.org/">downloading another browser</a> to get the best experience.
```
Comment (marcelgerber) [inline contribute.html]: These lines may need some wordsmithing.
Is it ok to link to the Google campaign http://www.whatbrowser.org?

**Comment by peterflynn** [comment]
How important is it to show a warning -- is brackets.io totally unusable with a non-modern browser?  If it's just minor visual glitches, I don't know if we really need to show a banner.  If we do show one, we should probably back off on the wording -- we're not a web app like Trello or GH where it'll be totally broken on an old browser.  And we don't want to give the impression that Brackets itself requires installing a newer browser in order to work...

**Comment by marcelgerber** [comment]
Yeah, that's true.
The most noticeable thing is probably that neither "New in Release ..." nor "Recent Blog Posts" will load, which may or may not confuse the user.
As stated before, the warning is only ever shown in very few browsers (keeping in mind that Brackets is a tech site).

**Comment by ficristo** [comment]
I have mixed feelings: I understand the reasoning but I don't think is really worth.

**Pull request closed without merging**